### DISCUSSION TITLE
CMAKE Tools Interpreter issue with Python 3.6 when building

### DISCUSSION THREAD

**Discussion by GitbyGit** [microsoft/vscode-cmake-tools#2280] [Q&A] [answered]
### Brief Issue Summary
I am trying the use Cmake tools in VS code to build the folder for the raspberry pi pico examples folder. However, when building, the following error occurs for python v3.6.

Extremely new to C development on raspberry pi, so bare with me.

<img width="1091" alt="Screen Shot 2021-12-05 at 9 01 26 PM" src="https://user-images.githubusercontent.com/35300870/144780316-e4ae4500-bbf7-4807-bb7e-a314f6dc66ff.png">

### CMake Tools Diagnostics

```
<!-- Paste the `CMake: Log Diagnostics` output HERE -->
```

### Debug Log

```
<!-- Paste the debug log contents HERE -->
```

**Comment by bobbrow** [comment, marked as answer]
Is that project supposed to install python for you, or are you expected to already have it present?  Have you tried running cmake in the terminal to configure this project?  Our extension attempts to automate the configuration using the same commands you would use in a terminal window, so this doesn't sound like an issue with our extension at first glance.  If configuring in a terminal works but the extension doesn't, then we have a bug on our hands.  I'm converting this issue to a discussion until we know for sure.

**Reply by GitbyGit** [reply]
I was able to resolve this issue. I was simply building the pico-examples library. I end up installing a new version of python 3 that worked.
